octoprint_mrbeam/software_update_information.py

```diff
@@ -366,13 +366,36 @@ def _generate_config_of_module(
                 tier=_get_tier_by_id(tier)
             )
 
-        if "update_script" in input_moduleconfig and module_id == "mrbeam":
-            update_script = os.path.join(
-                plugin._basefolder, "scripts", "update_script.py"
+        if "update_script" in input_moduleconfig:
+            if "update_script_relative_path" not in input_moduleconfig:
+                _logger.error(
```

Comment:
so you only need to log the error and continue normally ?

Reply:
User notification for update failed, right ?

Reply:
I added a notification `Error during fetching update information` with errorcode E-1003 and will break the fetching methode